horovod/common/operations.cc

```diff
@@ -1839,7 +1843,7 @@ bool RunLoopOnce(HorovodGlobalState& state, bool is_coordinator) {
           if (response.response_type() == new_response.response_type() &&
               response.devices() == new_response.devices() &&
               entry.tensor->dtype() == new_entry.tensor->dtype() &&
-              tensor_size + new_tensor_size <= state.tensor_fusion_threshold) {
+              tensor_size + new_tensor_size <= TensorFusionThresholdBytes()) {
```

Comment:
Should this be sourced from Parameter Manager?

Reply:
Yeah, I was just thinking about that.  That way we could also do some clever caching to avoid recomputing it every time as well.  I'll make that change.

Reply:
Nevermind, you already read Parameter Manager Threshold within TensorFusionThresholdBytes